libs/core/src/lib/datetime/public_api.ts

```diff
@@ -5,3 +5,4 @@ export * from './fd-date-formats';
 export * from './fd-datetime-adapter';
 export * from './fd-datetime.module';
 export * from './datetime-format.pipes';
+export * from './fd-date.utils';
```

Comment:
do we really need to keep it in the public?

Reply:
@dimamarksman yes, me and @N1XUS were struggling to find why `npm run build-pack-library` was failing for me and this is one of the things we found